### PR TITLE
fix: [CDS-68672]: modified the label

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger jest

**Comment by sahithikolichala** [comment]
retrigger cypress-cv

**Comment by sahithikolichala** [comment]
retrigger titlecheck

**Comment by sahithikolichala** [comment]
retrigger titlecheck

File: src/modules/72-triggers/strings/strings.en.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ addNewTrigger: Add New Trigger
 triggersSubLabel: 'All Trigger Types'
 noTriggersFound: 'No Triggers Found'
 newTrigger: + New Trigger
-selectStagesToExecute: Select Stages To Execute
+selectStagesToExecute: Pipeline Stages
```
Comment (mtran7) [inline src/modules/72-triggers/strings/strings.en.yaml:6]: should also update the key otherwise confusing

**Comment by sahithikolichala** [comment]
retrigger jest

File: src/modules/72-triggers/strings/strings.en.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ addNewTrigger: Add New Trigger
 triggersSubLabel: 'All Trigger Types'
 noTriggersFound: 'No Triggers Found'
 newTrigger: + New Trigger
-selectStagesToExecute: Select Stages To Execute
+selectPipelineStages: Pipeline Stages
```
Comment (mtran7) [inline src/modules/72-triggers/strings/strings.en.yaml:6]: Is it supposed to say Select Pipeline Stages or just pipeline Stages?

**Review by mtran7 — APPROVED** [review]

**Review by pankajsingh-harness — APPROVED** [review]

**Pull request merged**